content/docs/SUSHI/project/_index.md

```diff
@@ -3,96 +3,84 @@ title: "Project Structure"
 weight: 10
 ---
 
+{{% alert title="Warning" color="warning" %}}
+This documentation is for SUSHI 1.0.0, and this version contains significant changes to the project structure. For documentation on project structure in older versions of SUSHI, see the [legacy documentation](/docs/sushi/project-legacy).
+{{% /alert %}}
+
 ## Simple FSH Projects
 
-The simplest FSH project (sometimes referred to as a "FSH tank") contains only a configuration file and FSH files containing FHIR Shorthand definitions. For example, a simple FSH tank might look like this:
+The simplest FSH project (sometimes referred to as a "FSH tank") contains only a configuration file and an **input/fsh** subdirectory of FSH files containing FHIR Shorthand definitions. For example, a simple FSH tank might look like this:
 
 ```text
 simple-project
-├── config.yaml
-├── file1.fsh
-├── file2.fsh
-└── file3.fsh
+├── input
+|    └── fsh
+|        ├── file1.fsh
+|        ├── file2.fsh
+|        └── file3.fsh
+└── sushi-config.yaml
 ```
 
-The **config.yaml** file provides project configuration data to SUSHI. It is described further in the [Configuration](/docs/sushi/configuration/) documentation.
+The **sushi-config.yaml** file provides project configuration data to SUSHI. It is described further in the [Configuration](/docs/sushi/configuration/) documentation.
 
 Each FSH file can contain multiple FSH definitions of varying types. FSH file names are not significant, but must end with the **.fsh** extension. In addition, FSH files can be organized into subdirectories. This provides authors the flexibility to organize their FSH definitions in whatever way makes sense to then.
 
 {{% alert title="Tip" color="success" %}}
-A simple FSH project like the one shown above can be used with a minimal **config.yaml** file to create a bare-bones IG . It can also be used to generate only the FHIR resources by specifying the `FSHOnly` flag in **config.yaml**. Most authors who want to develop an IG using SUSHI, however, will also use an **ig-data** directory. Continue reading for more details.
+A simple FSH project like the one shown above can be used with a minimal **sushi-config.yaml** file to create a bare-bones IG . It can also be used to generate only the FHIR resources by specifying the `FSHOnly` flag in **sushi-config.yaml**.
 {{% /alert %}}
 
 ### Using a fsh Subdirectory with the HL7 IG Publisher and Auto-Builder
```

Comment:
I'm not sure about this header. Maybe the heading should just be "Using the HL7 IG Publisher and Auto-Builder" or maybe "Using an input/fsh Subdirectory..."?